src/components/chart/common-chart-options.js

```diff
@@ -144,6 +144,42 @@ class CommonChartOptions {
 
     getOptions = () => this.options;
 
+    getMobileOptions = (xAxisTickInterval, yAxisTickInterval) => {
+        return {
+            legend: {
+                itemStyle: {
+                    fontSize: this.constants.mobileFontSize,
```

Comment:
This merge has reinstated code that was previously removed - I will sort this out.

Reply:
Now sorted.